docs/contributing.md

```diff
@@ -38,7 +38,7 @@ Run `yarn lint:fix` if making JS/TS changes.
 If you made API changes, run `yarn api` to update the auto-generated API docs.
```

Comment:
I think we don't do this part anymore?

Reply:
Ok sounds good I'll take this out of the docs.